### ISSUE TITLE
How to deploy a new nixOS version?

### ISSUE THREAD

**Issue by francocalvo** [tweag/terraform-nixos#81]
Hey! 

I'm trying to deploy a new nixOS version to a GCE VM.

I've both modules, but they come with versions. 
It would be cool to have a complete example for a minimal deployment.

I'll tag the mantainer just in case: @adrian-gierakowski 
Thanks!

**Comment by adrian-gierakowski** [comment]
Hi, it's a while since I used this project, but can investigate. 
could you explain a bit more what you are having problem with?

**Comment by threddast** [comment]
@adrian-gierakowski are you now using something different to deploy NixOS with Terraform?

**Comment by adrian-gierakowski** [comment]
@threddast yes: https://github.com/nix-community/nixos-anywhere/tree/main/terraform

**Comment by threddast** [comment]
Wow, thank you! I looked at NixOS-anywhere before but it wasn’t clear that it would be helpful managing machines after first deployment

**Comment by adrian-gierakowski** [comment]
It works great. I use it together with https://github.com/nix-community/srvos and https://github.com/nix-community/disko
